themes/src/main/resources-community/theme/base/admin/messages/admin-messages_de.properties

```diff
@@ -0,0 +1,1513 @@
+#host-sending-registration-request-must-match.label=Host Sending Client Registration Request Must Match
+#host-sending-registration-request-must-match.tooltip=If on, then any request to Client Registration Service is allowed just if it was sent from some trusted host or domain.
+#client-uris-must-match.label=Client URIs Must Match
+#client-uris-must-match.tooltip=If on, then all Client URIs (Redirect URIs and others) are allowed just if they match some trusted host or domain.
+#allowed-protocol-mappers.label=Allowed Protocol Mappers
+#allowed-protocol-mappers.tooltip=Whitelist of allowed protocol mapper providers. If there is an attempt to register client, which contains some protocol mappers, which were not whitelisted, then registration request will be rejected.
+#consent-required-for-all-mappers.label=Consent Required For Mappers
+#consent-required-for-all-mappers.tooltip=If on, then all newly registered protocol mappers will automatically have consentRequired switch on. This means that user will need to approve consent screen. NOTE: Consent screen is shown just if client has consentRequired switch on. So it's usually good to use this switch together with consent-required policy.
+#allowed-client-scopes.label=Allowed Client Scopes
+#allowed-client-scopes.tooltip=Whitelist of the client scopes, which can be used on newly registered client. Attempt to register client with some client scope, which is not whitelisted, will be rejected. By default, the whitelist is either empty or contains just realm default client scopes (based on 'Allow Default Scopes' configuration property)
+#allow-default-scopes.label=Allow Default Scopes
+#allow-default-scopes.tooltip=If on, then newly registered clients will be allowed to have client scopes mentioned in realm default client scopes or realm optional client scopes
+#max-clients.label=Max Clients Per Realm
+#max-clients.tooltip=It won't be allowed to register new client if count of existing clients in realm is same or bigger than configured limit.
+
+#client-scopes=Client Scopes
+#client-scopes.tooltip=Client scopes allow you to define common set of protocol mappers and roles, that are shared between multiple clients
+
+groups=Gruppen
+
+group.add-selected.tooltip=Realm-Rollen die zu der Gruppen hinzugef\u00FCgt werden k\u00F6nnen.
+group.assigned-roles.tooltip=Realm Rollen die zur Gruppe zugeordnet sind
+#group.effective-roles.tooltip=All realm role mappings. Some roles here might be inherited from a mapped composite role.
+#group.available-roles.tooltip=Assignable roles from this client.
+#group.assigned-roles-client.tooltip=Role mappings for this client.
+#group.effective-roles-client.tooltip=Role mappings for this client. Some roles here might be inherited from a mapped composite role.
+
+default-roles=Standardrollen
+no-realm-roles-available=Keine Realmrollen verf\u00FCgbar
+
+users=Benutzer
+user.add-selected.tooltip=Realmrollen die dem Benutzer zugewiesen werden k\u00F6nnen.
```

Comment:
Here you use "Realmrollen", above you use "Realm-Rollen". I'm fine with both, just some consistence would be nice.